scripts/verify_opslevel_webhook_signature/go/go.mod

```
@@ -0,0 +1,3 @@
+module opslevelsignature
+
+go 1.20
```

```suggestion
go 1.23
```
Bumping go version to latest and tested that it still works. `1.20` is EOL'd [link](https://endoflife.date/go)